neps/nep-0171.md

```diff
@@ -248,6 +248,15 @@ interface NftMintLog {
     memo?: string
 }
 
+// An event log to capture token updating (e.g. new metadata)
+// Arguments
+// * `token_ids`: ["1", "abc"]
+// * `memo`: optional message
+interface NftUpdateLog {
+    token_ids: string[],
```

Comment:
Have we considered the option to include the updated NFT in the logs?
At the first glance, I see obvious pros - no need to make the request to see updated NFT, and obvious cons - the log is growing, while we are trying to save the letters. 
I don't have the answer here, just want to know your opinion.

Reply:
@telezhnaya It was discussed on the gov forum (see the link in the PR description). The major concern I see with posting more details into the logs is that the media might not fit into the logs. I think we are inevitably heading into the direction where we will use the logs to quickly catch if a relevant change happened, but then to learn more details you would need to execute some view function (as an optimization, we could even consider following the state updates on the indexer side, and execute view functions locally)

Reply:
Ah, I see. I totally forgot about the media, you are right.